client/app/components/navbar/navbar-notification.service.js

```diff
@@ -0,0 +1,109 @@
+class NavbarNotificationService {
+    constructor ($interval, $q, $translate, Alerter, constants, OvhApiNotificationAapi, translator, UNIVERSE) {
+        this.$interval = $interval;
+        this.$q = $q;
+        this.$translate = $translate;
+        this.alerter = Alerter;
+        this.constants = constants;
+        this.OvhApiNotificationAapi = OvhApiNotificationAapi;
+        this.translator = translator;
+        this.UNIVERSE = UNIVERSE;
+
+        this.NOTIFICATION_REFRESH_TIME = 60000;
+    }
+
+    getMessages () {
+        return this.$translate.refresh().then(() => this.OvhApiNotificationAapi.query({
+            lang: this.$translate.preferredLanguage(),
+            target: this.constants.target,
+            universe: this.UNIVERSE
+        }).$promise);
+    }
+
+    getSubLinks () {
+        return this.getMessages().then((messages) => messages.map((message) => this.convertSubLink(message))).catch(() => [{
+            id: "0",
+            date: moment(),
+            time: this._formatTime(moment()),
+            level: "error",
+            subject: this.translator.tr("common_navbar_notification_error_subject"),
+            description: this.translator.tr("common_navbar_notification_error")
+        }]);
+    }
+
+    /* eslint-disable class-methods-use-this */
+    _formatTime (dateTime) {
+        return moment(dateTime).fromNow();
+    }
+    /* eslint-enable class-methods-use-this */
+
+    _toggleSublinkAction (toUpdate, linkClicked) {
+        if (toUpdate.isActive && !toUpdate.updating) {
+            toUpdate.updating = true;
+            this.OvhApiNotificationAapi.post({ completed: [toUpdate.id] }).$promise.then(() => {
+                toUpdate.isActive = false;
+                toUpdate.acknowledged = true;
+            }).finally(() => { toUpdate.updating = false; });
+        } else if (!toUpdate.isActive && !toUpdate.updating && !linkClicked) {
+            toUpdate.updating = true;
+            this.OvhApiNotificationAapi.post({ acknowledged: [toUpdate.id] }).$promise.then(() => {
+                toUpdate.isActive = true;
+                toUpdate.acknowledged = true;
+            }).finally(() => { toUpdate.updating = false; });
+        }
+    }
+
+    convertSubLink (notification) {
```

Comment:
`Object.assign` but I'm not sure Babel for the Web Manager has a polyfill

Reply:
We don't have polyfill for the managers (dedicated inherit some polyfill from a non-ovh dependency, so it don't count). Of curse I could use lodash to redifine my Object, but there's no guideline about it and the result would be exactly the same.